modules/client-configuration/pages/contact-methods-saltssh-tunnel.adoc

```diff
@@ -34,18 +40,21 @@ To overwrite it, you can add a custom port numbers greater than 1024 to [path]``
 ssh_push_port_https = high_port
 ----
 
-After changing this configuration parameter you must restart services
-such as [systemitem]``salt-secrets-config.service``,
-[systemitem]``tomcat.service``, and
-[systemitem]``taskomatic.service``.  To cover all needed services, it
-is the best to restart [command]``spacewalk-service`` as root:
+After changing this configuration parameter you must restart services such as [systemitem]``salt-secrets-config.service``, [systemitem]``tomcat.service``, and
+[systemitem]``taskomatic.service``.
+To cover all needed services, it is the best to restart the {productname} Server as root:
 
 ----
-spacewalk-service restart
+mgradm restart
 ----
 
 
 
+[IMPORTANT]
+====
+Use [literal]``mgrctl term`` before running steps inside the server container.
+====
```

Comment:
The key is to say which steps need to run inside of the container and which not. This sentence expect that the customer know that need to be executed where.

And here the "sudo" configuration happens on the client. This is not in a container